### ISSUE TITLE
Make it easier to show the sidebar

### ISSUE THREAD

**Issue by SimpleSamples** [Microsoft/vscode#64876]
I hid my sidebar and can't find it. It should be easier to find. I looked for things like "_Toggle Sidebar_" but I can't find anything like that. I searched for answers and I can't find anything specifying how to get the sidebar back.

I think there should be a "Toggle Sidebar" in the View menu.

**Comment by mjbvz** [comment]
These do exist in the menu and in the command palette:

![screen shot 2018-12-12 at 11 41 39 am](https://user-images.githubusercontent.com/12821956/49894505-ee86c700-fe02-11e8-9d4e-5cffc09b3f2e.png)
![screen shot 2018-12-12 at 11 41 26 am](https://user-images.githubusercontent.com/12821956/49894506-ee86c700-fe02-11e8-9809-ea56276ab500.png)

**Comment by SimpleSamples** [comment]
I used the wrong term. I don't know what the correct term is, it is gone so I can't find what to call it. It is more of a toolbar than a sidebar. It was on the right side and had icons for things like showing the explorer. I have the Bookmarks extension and there was an icon for it.

Okay, I found it. It is the Activity Bar. I don't know how I turned it off but it was too easy to turn it off compared to finding it to turn it back on.
